src/Console/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -322,7 +322,7 @@ protected function commandStringToArgs(string $command): array
 
 // phpcs:disable
 class_alias(
-    'Cake\Console\TestSuite\ConsoleIntegrationTestTrait',
+    \Cake\Console\TestSuite\ConsoleIntegrationTestTrait::class,
```

Comment:
Can we keep the class alias with old way? Then you dont need to fix the validation script

Reply:
Only in class_alias in tests?

Reply:
Seems like this is checked by the script which so far expects the normal string
This FQCN would be invalid anyway, since it would have to be normalized into a use statement anyway if included with ::class syntax